packages/paste-website/src/pages/core/changing-theme.mdx

```diff
@@ -0,0 +1,90 @@
+---
+title: Changing the Theme
+description: When using Paste, migrating to a different theme is a simple two step process.
+slug: /core/changing-theme
+---
+
+import {graphql} from 'gatsby';
+import {SidebarCategoryRoutes} from '../../constants';
+
+export const pageQuery = graphql`
+  {
+    mdx(fields: {slug: {eq: "/core/changing-theme/"}}) {
+      fileAbsolutePath
+      frontmatter {
+        slug
+        title
+      }
+      headings {
+        depth
+        value
+      }
+    }
+  }
+`;
+
+<ComponentHeader
+  name={props.pageContext.frontmatter.title}
+  description={props.pageContext.frontmatter.description}
+  categoryRoute={SidebarCategoryRoutes.CORE}
+/>
+
+---
+
+ <contentwrapper>
+
+<PageAside data={props.data.mdx} />
+
+<content>
+
+## Update the theme provider
+
+The `<Theme.Provider>` is a wrapper component that provides a theme object to any descendant. It is usually towards the root of your app.
+
+```jsx
+import {Theme} from '@twilio-paste/theme';
+
+<Theme.Provider theme="default">
+  <App />
+</Theme.Provider>;
+```
+
+To change the theme, simply change the value of the `theme` prop on the `Theme.Provider`. If you are using Paste component correctly and only usign the tokens to style your app, the `<Theme.Provider>` handles (almost) everything for you.
```

Comment:
> "If you are using Paste component correctly and only usign.."

This reads a bit weird and there's a typo for 'using'. Drop 'component' or change to 'components'?

Reply:
Updated! Now it says 
> If you are using Paste components correctly and only using the tokens to style your app, the <Theme.Provider> handles (almost) everything for you.